Demo/AuthenticatorDemo/Secrets.example.swift

```diff
@@ -0,0 +1,14 @@
+import Foundation
+
+@objc
```

Comment:
I didn't check, but seeing there are only a few Objective-C files in this repo makes me wondering if this `@objc` is needed.

Reply:
Addressed in [wordpress-mobile/WordPressAuthenticator-iOS@`487132f` (#706)](https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/706/commits/487132f9a642e07e9c6c36337133f19c50108d4b)